lib/home_page.dart

```diff
@@ -125,6 +127,39 @@ class HomePageState extends State<HomePage> {
     controller.animateCamera(CameraUpdate.newCameraPosition(CameraPosition(
         target: LatLng(lat, long), zoom: 15, tilt: 50.0, bearing: 45.0)));
   }
+
+  Widget _zoomminusfunction() {
+
+    return Align(
+      alignment: Alignment.topLeft,
+      child: IconButton(
+          icon: Icon(Icons.zoom_out,color:Colors.orange),
+          onPressed: () {
+            zoom--;
+            _minus( zoom);
+          }),
+    );
+  }
+  Widget _zoomplusfunction() {
+
+    return Align(
+      alignment: Alignment.topRight,
+      child: IconButton(
+          icon: Icon(Icons.zoom_in,color:Colors.orange),
+          onPressed: () {
+            zoom++;
+            _plus(zoom);
+          }),
+    );
+  }
+  Future<void> _minus(double zoomVal) async {
+    final GoogleMapController controller = await _controller.future;
+    controller.animateCamera(CameraUpdate.newCameraPosition(CameraPosition(target: LatLng (55.676098, 12.568337), zoom: zoomVal)));
```

Comment:
should be dependant on current camera position?

Reply:
Please fix your conflict before to merge with master.